bank.py

```diff
@@ -1,19 +1,21 @@
 from user import User
 from typing import Optional
-
+import json
 
 class ATM:
     def __init__(self):
         self.users = []
+        self.users_JSON = []
         self._read_users()
 
     def _read_users(self):
-        """reads file and writes users"""
-        with open("database.txt", "r") as f:
-            lines = f.readlines()
-            for line in lines:
-                info = line.split(":")
-                user = User(info[0], info[1], int(info[2]))
+        """reads file.json and writes users"""
+        with open("database.json") as file:
+            data_users = json.load(file)
+            for data in data_users["users"]:
+                user = User(data["login"],
+                            data["pass"],
+                            data["balance"])
                 self.users.append(user)
```

Comment:
what happens if we call _read_users a few times?
We will have a duplication of users in the list. I know that this logic was before but let's fix this.

Do:
Rewrite this method, it should return list of users instead of adding to list.
in line 9 do something like users = _read_users()